var/spack/repos/builtin/packages/py-tensorflow-probability/package.py

```diff
@@ -31,7 +38,10 @@ class PyTensorflowProbability(Package):
         url="https://github.com/tensorflow/probability/archive/0.8.0.tar.gz",
```

Comment:
I vote we deprecate this version. It's ancient and has a custom URL syntax that makes the package look ugly.

Reply:
This is why we have a deprecation policy. We aren't deleting it immediately, just deprecating it for a full release cycle before removing. https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions describes the policy in more detail.

I don't have strong opinions about this, and I don't mind keeping it if anyone still uses it.